app/script/conversation/ConversationRepository.js

```diff
@@ -2291,9 +2294,20 @@ z.conversation.ConversationRepository = class ConversationRepository {
                   ? z.string.modalConversationNewDeviceHeadlineYou
                   : z.string.modalConversationNewDeviceHeadlineOne;
               } else {
-                const log = `Granting '${consentType}' message '${messageType}' in '${conversationId}' needs user ids`;
+                const log = `Granting message '${messageType}' in '${conversationId}' for '${consentType}'  needs ids`;
```

Comment:
`needs ids` -> `needs user ids`?

Reply:
Need to reword this. It no longer fits into a single line if I add the word user with prettier. The message is obviously not concise enough in that case.

Reply:
There are also 2 spaces before `needs`.